examples/todomvc/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+authors = ["Nick Fitzgerald <fitzgen@gmail.com>"]
+edition = "2018"
+name = "dodrio-todomvc"
+version = "0.1.0"
+
+[dependencies]
+cfg-if = "0.1.6"
+futures = "0.1"
+serde_json = "1.0.38"
+wasm-bindgen = "0.2"
+wasm-bindgen-futures = "0.3"
+
+[dependencies.console_error_panic_hook]
+optional = true
+version = "0.1.1"
+
+[dependencies.console_log]
+optional = true
+version = "0.1.2"
+
+[dependencies.dodrio]
+path = "../.."
+
+[dependencies.log]
+optional = true
+version = "0.4.6"
+
+[dependencies.serde]
```

Comment:
I might recommend collapsing these using inline tables where possible (as it's a bit more idiomatic I think)

Reply:
I updated my `cargo update` but it still is canonicalizing everything like this :(

Reply:
Yeah I don't think they're using something which preserves formatting (not sure if that exists in a production-ready capacity even...)